File: test/style_tests/fib.js
function fib(n) {
  if (n < 0) {
    throw "The argument to fib must be non-negative";
  } else if (n <= 1) {
    return n;
  } else {
    return fib(n - 1) + fib(n - 2);
  }
}
console.log(fib(0));
console.log(fib(1));
console.log(fib(2));
console.log(fib(3));
console.log(fib(4));
console.log(fib(5));
console.log(fib(6));

File: test/style_tests/fib.scala
def fib(n : Int) =
  if (n < 0) {
    throw "The argument to fib must be non-negative"
  } else if (n <= 1) {
    n
  } else {
    fib(n - 1) + fib(n - 2)
  }
println(fib(0))
println(fib(1))
println(fib(2))
println(fib(3))
println(fib(4))
println(fib(5))
println(fib(6))